_includes/configuration-db.md

```diff
@@ -36,7 +36,9 @@ disks; installation will fail if the disks are not blank.
         All {{ site.data.product.title_short }} appliances in a multi-region
         deployment must use the same key.
 
-6.  Choose **Create Internal Database** for the database location.
+6.  Choose **Create Internal Database** for the database location. 
+
+    In the **Configure Messaging** menu when you select **Make No messaging changes**, then the ```Configuration failed: Internal database require a volume mounted at /var/lib/pgsql. Please add an unpartitioned disk and try again.``` message is displayed.
```

Comment:
@agrare Would you please reshare the [above](https://github.com/ManageIQ/manageiq-documentation/pull/1691/files#diff-e6503c312761f591c879d6d8313cb6a4db6bf696803dd65d500e104b4d59635aR5) link because when I click, it's taking me to the same comment that you mentioned.

Reply:
@meghasalunke1 it points to line 5 on this file: `Before installing an internal database, add a disk to the infrastructure
hosting your appliance.`

Reply:
@meghasalunke1 could you please make the change addressing this?

Reply:
Added changes, @Fryguy I couldn't give the reference to the same page, would you please help, instead of adding 'above', I want to provide link for 'Configuring an Internal Database' (Title).